igcollect/switch.py

```diff
@@ -328,6 +355,23 @@ def ports_stats(prefix, snmp, ports, model):
                 print(template.format(port_name, counter, data))
 
 
+def dom_stats(prefix, snmp, ports, model):
+    """
+    Print graphite-compatible Digital Optical Monitoring stats for each port
+    of the switch
+    """
+
+    for metric, oid in DOM_OIDS[model].items():
```

Comment:
From the scope of the function it can happen that `DOM_OIDS[model]` will result in a KeyError.

You could avoid it altogether with @Woellchen 's suggestion in his [comment](https://github.com/innogames/igcollect/pull/171/files/1ab699a4271f8e35fd7d02b664e08a1023a75ed4#r944546214).
Or thinking about what should happen if a model is not in `DOM_OIDS`.

Reply:
I went with Patrick's suggestion.